docs-ref-services/keyvault.md

```diff
@@ -15,96 +15,44 @@ ms.service: key-vault
 
 # Azure Key Vault libraries for Java
 
-## Overview
+The Azure Key Vault client libraries for Java offer a convenient interface for making calls to Azure Key Vault. For more information about Azure Key Vault, see [Introduction to Azure Key Vault](https://docs.microsoft.com/azure/key-vault/general/overview).
 
-Safeguard and manage cryptographic keys and secrets used by cloud applications and services with [Azure Key Vault](/azure/key-vault/).
+## Libraries for data access
 
-To get started with Azure Key Vault, see [Get started with Azure Key Vault](/azure/key-vault/key-vault-get-started).
+The latest version of the Azure Key Vault libraries is version 4.x. Microsoft recommends using version 4.x for new applications.
 
-## Client libraries
+If you cannot update existing applications to version 4.x, then Microsoft recommends using version 1.x.
 
-Create, update, and delete keys and secrets in Azure Key Vault with the client libraries.
-We offer separate Client libraries to manage Secrets and Keys in your Key Vault. Azure Key Vault Secrets client library allows you to securely store and tightly control the access to tokens, passwords, API keys, and other secrets. 
-Azure Key Vault Keys client supports RSA keys and elliptic curve keys, each with corresponding support in hardware security modules (HSM). Multiple keys, and multiple versions of the same key, can be kept in the Key Vault. Cryptographic keys in Key Vault are represented as JSON Web Key [JWK] objects. 
+### Version 4.x
 
+The version 4.x client libraries for Java are part of the Azure SDK for Java. The source code for the Azure Key Vault client libraries for Java is available on [GitHub](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/keyvault).
 
-[Add a dependency](https://maven.apache.org/guides/getting-started/index.html#How_do_I_use_external_dependencies) to your Maven `pom.xml` file to use the client library in your project.  
+Use the following version 4.x libraries to work with certificates, keys, and secrets:
 
-```XML
-<dependency>
-    <groupId>com.azure</groupId>
-    <artifactId>azure-security-keyvault-secrets</artifactId>
-    <version>4.0.0</version>
-</dependency>
+| Library | Reference | Package | Source |
+|----------------------------------------|-------------------------------------------------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+|    azure-security-keyvault-certificates   |    [Reference](https://azuresdkdocs.blob.core.windows.net/$web/java/azure-security-keyvault-certificates/4.1.0-beta.2/index.html)    |    [Maven](https://search.maven.org/artifact/com.azure/azure-security-keyvault-certificates)    |    [GitHub](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/keyvault/azure-security-keyvault-certificates) |
```

Comment:
Is there no way to point to the latest each time? We really don't even want this pointing to beta versions, but release versions. Someone will have to update this every time we ship, since we currently have no automation in place. This will be hard to sustain.

/cc @g2vinay @alzimmermsft

Reply:
Yeah, this should point to latest stable.
@JonathanGiles @srnagar

Reply:
Do you have any doc endpoints that don't even require a version? For .NET and Java we do (for all URLs, since nuget.org/packages/<package ID> works) but I know Python has to use at least version-specific URLs to pypi.org or something like that).